runtime-decompiler/src/main/java/org/jrd/backend/data/ArchiveManagerOptions.java

```diff
@@ -0,0 +1,23 @@
+package org.jrd.backend.data;
+
+import java.util.ArrayList;
+
+public class ArchiveManagerOptions {
+	private static ArrayList<String> extensions = new ArrayList<>();
```

Comment:
Except defaults, nothing should be static. I would even avoid add and clear. Simple set (and get yo already have) is enough

Reply:
The ArchiveManagerOptions should be singleton, with  normla, object fields (except defaults) instead of statics. Statics are for constants and utility methods only. It is always goal, to move as far from static as possible.